ui/src/components/homePage/homePage/popups/registration.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@Core/component';
+
+export class Registration extends Component {
+    private LOCATORS = {
+        registrPopupTitle: this.locator.locator('//h2[contains(@class, "loginPopup")]'),
+        registrCreateButton: this.locator.locator('//button[@aria-label="Create new account"]'),
```

Comment:
You can remove preffix registr for these locators
the meaning will not change, but the code will be clean and readable
this.LOCATORS = {
  title: this.locator(...),
  buttonCreate: this.locator(...)
  }